microshift_troubleshooting/microshift-troubleshoot-backup-restore.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: ASSEMBLY
+[id="microshift-troubleshoot-backup-restore.adoc"]
+= Troubleshoot data backup and restore
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-troubleshoot-data-backup-and-restore
+
+toc::[]
+
+To troubleshoot failed data backups and restorations, check the basics first, such as data paths, storage configuration, and storage capacity.
+
+[id="troubleshoot-backup-restore-microshift-backup-data-failed"]
+== Backup data failed
+Data backups are automatic on OSTree systems. If you are not using an OSTree system and attempted to create a manual backup, the following reasons could cause the backup to fail:
+
+* If {product-title} stopped running because of an error, you cannot perform a backup of the data.
+** First, make sure the system is healthy.
+** Then, stop it in a healthy state before attempting a backup.
+** You might need to wait several minutes after a system start to successfully stop {product-title}. Allow the system to complete health checks and any other background processes before attempting to back up.
+
+* If you do not have sufficient storage for the data, the backup fails. Ensure that you have enough storage for the {product-title} data.
+
+* If you do not have sufficient permissions, a backup can fail. Ensure you have the correct user permissions to create a backup and perform the required configurations.
+
+[id="troubleshoot-backup-restore-microshift-backup-logs"]
+== Backup logs
+* With manual backups, the logs print to the console.
+* For RPM-OSTree automated backups, logs are automatically generated. You can check OSTRee logs by running the following command:
```

Comment:
Can we say something like "backup logs are part of MicroShift service logs (journal)"?
"You can check OSTRee logs" - I'd change OSTRee for MicroShift